### PR TITLE
Add Sphinx site and shields; update CI Pipeline

### PR DESCRIPTION
**Pull request by umarcor**
This PR brings a minimal plumbing to this repo.

The main motivation is to have some site on GitHub Pages, which we can cross-reference from [edaa-org.github.io](https://edaa-org.github.io) instead of having broken links.
By the way, a "work in progress" sign is added both in the README and in the index of the site.

A subset of the Pipeline from pyTooling/Actions is used to replace the CI jobs. Unit testing and coverage are disabled because this repo does not have tests yet.

### CODE REVIEW

File: .github/workflows/Pipeline.yml [hunk 1]
```diff
@@ -1,274 +1,106 @@
...
-          - {python: 3.6, icon: 🔴}
-          - {python: 3.7, icon: 🟠}
-          - {python: 3.8, icon: 🟡}
-          - {python: 3.9, icon: 🟢}
+  Params:
+    uses: pyTooling/Actions/.github/workflows/Parameters.yml@r0
+    with:
+      name: pyEDAA.CLITool
+      python_version: '3.9'
+      python_version_list: '3.6 3.7 3.8 3.9'
```
Comment (Paebbels) [inline .github/workflows/Pipeline.yml:18]: Please use default and also enable 3.10.

File: .github/workflows/Pipeline.yml [hunk 2]
```diff
@@ -1,274 +1,106 @@
...
+      - BuildTheDocs
+      - PublishToGitHubPages
+    with:
+      package: ${{ fromJson(needs.Params.outputs.params).artifacts.package }}
+      remaining: |
+        ${{ fromJson(needs.Params.outputs.params).artifacts.doc }}
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.6
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.7
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.8
+#        ${{ fromJson(needs.Params.outputs.params).artifacts.unittesting }}-3.9
```
Comment (Paebbels) [inline .github/workflows/Pipeline.yml:105]: Also add 3.10 here.

Comment (umarcor) [reply .github/workflows/Pipeline.yml:104]: Done. However, note that it is not used (until tests are added).

File: .github/workflows/Pipeline.yml (hunk 1 above)
Comment (umarcor) [reply .github/workflows/Pipeline.yml]: Done.

**Pull request merged**